packages/contracts-bedrock/scripts/checks/check-interfaces.sh

```diff
@@ -179,8 +179,23 @@ for interface_file in $JSON_FILES; do
     interface_abi=$(jq '[.abi[] | select(.type != "constructor")]' < "$interface_file")
     contract_abi=$(jq '[.abi[] | select(.type != "constructor")]' < "$corresponding_contract_file")
 
+    # Function to normalize ABI by replacing interface name with contract name
+    # Base contracts aren't allowed to use interfaces (guarantees a 1:1 match)
```

Comment:
just trying to clarify this, is this what you mean? 
```suggestion
    # Base contracts aren't allowed to inherit from their interfaces (guarantees a 1:1 match)
```